insecure-db-2.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_db_instance" "default2" {
+  allocated_storage    = 10
+  engine               = "mysql"
+  engine_version       = "5.7"
+  instance_class       = "db.t3.micro"
+  name                 = "mydb"
+  username             = "foo"
+  password = "foobarbaz"
+  parameter_group_name = "default.mysql5.7"
+  skip_final_snapshot  = true
+  publicly_accessible = true
```

Comment:
### :warning: Aqua detected misconfiguration in your code  
**Misconfiguration ID:** AVD-AWS-0180   
**Check Name:** RDS Publicly Accessible   
**Severity:** HIGH   
**Message:** Instance has Public Access enabled  
**Resolution:** Remove the public endpoint from the RDS instance'  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/misconfig/n/a